ReactWindows/ReactNative.Net46/Modules/NetInfo/NetInfoModule.cs

```diff
@@ -67,15 +69,15 @@ public void OnDestroy()
         public void OnResume()
         {
             _networkInfo.Start();
-            _networkInfo.NetworkAvailabilityChanged += OnAvailabilityChanged;
+            _networkInfo.NetworkAddressChanged += OnAddressChanged;
```

Comment:
NetworkStatusChanged seems like a better choice if you aren't going to provide separate events for availability and address changed.